live/src/config.sh

```diff
@@ -164,6 +164,34 @@ rm -rf /usr/share/doc/packages/*
 du -h -s /usr/share/man
 rm -rf /usr/share/man/*
 
+# python is installed just because of few simple scripts /usr/sbin/bcache-status (bcache-tools package)
+# and /usr/sbin/xfs_protofile (xfsprogs package)
+# both are not used by libtorage-ng, yast2-storage-ng nor agama
+python=$(rpm -q --whatprovides python3-base || true)
+if [ -n "$python" ]; then
+  echo "Python package: $python"
+  python_deps=$(rpm -e "$python" 2>&1 || true)
+  # avoid removing python accidentally because of some new unknown dependency
+  python_deps=$(echo "$python_deps" | grep -v -e "Failed dependencies" -e "needed by .* libpython" -e "needed by .* bcache-tools" -e "needed by .* xfsprogs" || true)
+
+  if [ -z "$python_deps"]; then
+    echo "Removing Python..."
+    # remove libpython as well
+    rpm -e --nodeps "$python" $(rpm -qa | grep "^libpython3")
+  else
+    echo "Warning: Extra Python dependency detected:"
+    echo "$python_deps"
+    echo "Keeping the python packages installed"
+  fi
+fi
+
+# remove OpenGL support
+rpm -qa | grep ^Mesa | xargs rpm -e --nodeps
+
+# uninstall libyui-qt and libqt (pulled in by the YaST dependencies)
```

Comment:
I think agama web ui should simply provide libyui backend, so it is not installed